### PR TITLE
Add eta-based info to quoting

### PR DESCRIPTION
**Pull request by taryneast**
Relevant trello card:
https://trello.com/c/oU7rLj3C

Actual API PR:
https://github.com/sendle/sendle/pull/2896

Changes made:
Add eta-section to all sample responses.
Add field-descriptions for all response fields (previously not present) including describing eta-fields
Shuffle examples etc so they align a bit better with the neighbouring text.

### CODE REVIEW

**Comment by taryneast** [comment]
[ETAs in Quotes](https://trello.com/c/oU7rLj3C/3-etas-in-quotes)

**Comment by kiesia** [comment]
@taryneast do you mind posting a screenshot of the update? It's pretty hard to follow just in code. 🙂

**Comment by taryneast** [comment]
Yup api-doc changes are hard to test as we don't have a PR-deployed version :P 

Ok, here is what it looks like now. You can look at existing api-doc to see what it used to look like.

screen shot 2018-02-16 at 10 57 44 am
screen shot 2018-02-16 at 10 58 00 am
screen shot 2018-02-16 at 10 58 11 am
screen shot 2018-02-16 at 10 58 21 am
screen shot 2018-02-16 at 10 58 29 am
screen shot 2018-02-16 at 10 58 44 am
